Formula/pre-commit.rb

```diff
@@ -47,7 +47,16 @@ def post_install
             -   id: trailing-whitespace
       EOS
       system bin/"pre-commit", "install"
-      system bin/"pre-commit", "run", "--all-files"
+      (testpath/"f").write "hi\n"
+      system "git", "add", "f"
+
+      ENV["GIT_AUTHOR_NAME"] = "test user"
+      ENV["GIT_AUTHOR_EMAIL"] = "test@example.com"
+      ENV["GIT_COMMITTER_NAME"] = "test user"
+      ENV["GIT_COMMITTER_EMAIL"] = "test@example.com"
+      git_exe = which('git')
```

Comment:
needs to be " not ' or style won't be happy

Reply:
oops! opposite from python lol

Reply:
It's just a rubocop setting. Cask still does the single-where-no-interpolating thing https://github.com/caskroom/homebrew-cask/blob/master/Casks/java.rb

Reply:
ah makes sense 👍